zh/editor/node-library/index.md

```diff
@@ -0,0 +1,70 @@
+# 节点预制库
+
+**节点预制库** 提供便捷的管理需要复用的节点，提供 **内置** **自定义** 列表
+
+## 使用方式
+
+- 拖动列表上的节点到 **层级面板** 或 **场景面板** 生成新节点。
+- 拖动 **资源面板** 上的 prefab 预制件资源到 **自定义** 列表，便收藏了一个新的自定义节点。
+- **自定义** 列表的节点支持右击菜单：删除，定位到资源。
+
+
+## 扩展面板列表
+
+走插件的 "contributions" 扩展方式
+
+package.json
+
+``` json
```

Comment:
```json5
```json5
```

Reply:
@pandamicro said just json.... not json5.